meson.build

```diff
@@ -1,5 +1,5 @@
 project('rizin', 'c',
-  version: 'v0.4.0-git',
+  version: 'v0.4.1',
```

Comment:
Is this change necessary?

Reply:
no, not this one. `v0.4.0` is necessary though